### DISCUSSION TITLE
PIS e COFINS em NF modelo 65

### DISCUSSION THREAD

**Discussion by Mschirmann** [TadaSoftware/PyNFe#357] [General]
Eai pessoal.. vi que antigamente a validação de PIS e COFINS funcionava para qualquer modelo (55,65) porém após algumas atualizações no código agora só realiza a serialização de dados de PIS/COFINS para NFs 55.
Saberiam me dizer se é tranquilo implementar para modelo 65 ou se teria que alterar muita coisa?

**Comment by leogregianin** [comment]
Teria que implementar isso. Vou transferir essa discussão para uma issue.

**Comment by leogregianin** [comment]
A regra do PIS/COFINS para NFCe (modelo 65) ficou:
* A modalidade (CST) sempre vai ser obrigatória.
* Se não informar ou informar zero para o campo `pis_valor_base_calculo`, `pis_aliquota_percentual` e `pis_valor` e a modalidade não for "04", "05", "06", "07", "08", "09", "49", "99" não vai gerar o grupo.
